editors/emacs/lambdapi-proofs.el

```diff
@@ -9,9 +9,26 @@
 
 (defconst lp-goal-line-prefix "---------------------------------------------------")
 
+
+(defun lp-focus-goal (goalno &optional proofbuf)
```

Comment:
Could you please document this function, following approximately <https://github.com/bbatsov/emacs-lisp-style-guide#docstrings>?